src/dvclive/live.py

```diff
@@ -333,6 +348,24 @@ def log_artifact(self, path: StrPath):
                 logger.warning(f"Failed to dvc add {path}: {e}")
                 return
 
+            if type == "model":
```

Comment:
We could, I don't know if we should. 
My motivation was to not include unnecessary/unrelated things in the P.R.

It would not be that hard to extend in the future if/when we need it.

Reply:
I guess I worry it's harder to document this way. How do you explain what the other artifact metadata args do?

Not a blocker.

Reply:
> How do you explain what the other artifact metadata args do?

Write to `.dvc` file?

Reply:
Side note. 

`artifacts` section has a restriction on `names` formatting that is only applicable because of the model registry + `git tag` usage.

It feels harder to document that we would try to write to `artifacts` except if the `name` we found or the one you provide doesn't fulfill a restriction that doesn't apply to your use case.

Reply:
> They are written to .dvc file? (I updated the P.R. to do that)

I don't think we should have two different places to add these fields. I'd rather deprecate putting them into `.dvc` files like we are doing with `artifacts.yaml`. This feels like we are only partially committing to the new MR workflow. Is there some hesitation to rely on this new `artifacts` section?

> It feels harder to document that we would try to write to `artifacts` except if the `name` we found or the one you provide doesn't fulfill a restriction that doesn't apply to your use case.

What other use cases do you have in mind?